srv/modules/runners/validate.py

```diff
@@ -413,9 +413,13 @@ def _monitor_check(self, name):
             self.errors.setdefault(name, []).append(msg)
         elif same_hosts:
             count = len(same_hosts.keys()[0].split(","))
-            if count < 3:
-                msg = "Must have at least three entries"
-                self.errors[name] = [ msg ]
+            if (not self.in_dev_env and count < 3) or (self.in_dev_env and count < 1):
+                if self.in_dev_env:
+                    msg = "Must have at least one monitor"
+                    self.errors[name] = [ msg ]
```

Comment:
Move this line to after the "if" block, and remove the same line in the "else" branch.